.azure-pipelines/cd-build-deploy-python.yml

```diff
@@ -0,0 +1,199 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Build, package, and deploy Python packages for microsoft_agents_m365copilot_core and microsoft_agents_m365copilot_beta
+
+name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.r)
+
+trigger:
+  branches:
+    include:
+      - main
+  paths:
+    include:
+      - 'python/packages/microsoft_agents_m365copilot_beta/generated/**'
+      - 'python/packages/microsoft_agents_m365copilot_beta/src/agents_m365_copilot_beta_request_adapter.py'      
+      - 'python/packages/microsoft_agents_m365copilot_beta/src/agents_m365_copilot_beta_service_client.py'
+      - 'python/packages/microsoft_agents_m365copilot_core/src/**'
+  tags:
+    include:
+      - '*microsoft_agents_m365copilot_core'
```

Comment:
same comment about the duplicated trigger

Reply:
That's okay, I've added conditions to the tasks to limit which tasks are run based on the tags added by RP. Probably would be better to make separate pipelines for each library, will address that later.